[2022-02-09 08:59:41,658] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-11-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:41,702] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-11-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:41,702] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:41,702] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:41,702] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:41,739] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-11-01 23:00:00+00:00
[2022-02-09 08:59:41,765] {standard_task_runner.py:52} INFO - Started process 296 to run task
[2022-02-09 08:59:41,776] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-11-01T23:00:00+00:00', '--job-id', '475', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpq1ykryti', '--error-file', '/tmp/tmp7lpdd8hp']
[2022-02-09 08:59:41,778] {standard_task_runner.py:77} INFO - Job 475: Subtask local_to_gcs_task
[2022-02-09 08:59:41,891] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-11-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:41,996] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:42,044] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-11-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-11-01T23:00:00+00:00
[2022-02-09 08:59:49,665] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 08:59:49,680] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20201101T230000, start_date=20220209T085941, end_date=20220209T085949
[2022-02-09 08:59:49,738] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:49,780] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:14:45,018] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-11-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:45,047] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-11-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:45,047] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:45,047] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:14:45,047] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:45,066] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-11-01 23:00:00+00:00
[2022-02-09 09:14:45,074] {standard_task_runner.py:52} INFO - Started process 330 to run task
[2022-02-09 09:14:45,081] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-11-01T23:00:00+00:00', '--job-id', '562', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmps_pn52sz', '--error-file', '/tmp/tmp2h9g90n5']
[2022-02-09 09:14:45,082] {standard_task_runner.py:77} INFO - Job 562: Subtask local_to_gcs_task
[2022-02-09 09:14:45,177] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-11-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:14:45,241] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:45,275] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-11-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-11-01T23:00:00+00:00
[2022-02-09 09:14:52,324] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:14:52,490] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20201101T230000, start_date=20220209T091445, end_date=20220209T091452
[2022-02-09 09:14:52,589] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:52,702] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
